Trappist/src/Promact.Trappist.Repository/TestSettings/TestSettingsRepository.cs

```diff
@@ -8,7 +9,7 @@ namespace Promact.Trappist.Repository.TestSettings
     public class TestSettingsRepository : ITestSettingsRepository
     {
         private readonly TrappistDbContext _dbContext;
-        Test test = new Test();
+
```

Comment:
please update UpdateTestSettings method. use auto mapper and return updated object.